distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -301,6 +315,15 @@ public void tearDown() throws Exception {
 
         cswServer.stop();
 
+        //waits until the downloads are empty
```

Comment:
We probably don't need both, the comment and the debug log. Actually, the `expect()` pretty much says what we're waiting for so we probably don't need either.